src/styles/navbar.css

```diff
@@ -16,6 +16,7 @@
 
 .nav-bar li {
   list-style: none;
+  text-transform: none;
```

Comment:
- [ ] I cannot find many of the styles that responsible for your design in PR. Be that as it may, I have decided to pass my comment across to you here. It is part of the project requirements to produce the exactly replica of the Zeplin desing, but your design is absolutely different from the expected design. Kindly  check the [project requirements]() to re-confirm your expectation for this milestone. Below are the designs for the Zeplin and your design. Work your design alongside the zeplin design.

### Project Requirements
![project-rqt](https://user-images.githubusercontent.com/26597432/230715673-f93bc1ec-b94d-405f-9b23-5b3c38345474.png)

### Your Current Design
![your-design](https://user-images.githubusercontent.com/26597432/230715677-e46d2b65-c996-4223-9585-d0beb13bf812.png)

### Zeplin Design
![zeplin-design](https://user-images.githubusercontent.com/26597432/230715685-ed133b0e-b0cd-43a0-9c60-63574a836a9a.png)